README.md

```diff
@@ -1,140 +1,127 @@
-# govcookiecutter
-A cookiecutter template for data science projects within UK Government.
+# `govcookiecutter`
 
-This repository replaces the original
-[`cookiecutter-data-science-gds`](https://github.com/ukgovdatascience/cookiecutter-data-science-gds), which has been archived.  Please start using this one.
+A cookiecutter template for analytical, code-based projects within Her Majesty's Government.
```

Comment:
Did we make any progress on thinking to whether this is HM Government or wider public-sector?

Reply:
It's probably applicable to wider public-sector (or anyone for that matter!), but I'll leave this here for now!